### ISSUE TITLE
Cannot call method 'create' of undefined

### ISSUE THREAD

**Issue by justan0therdave** [versolearning/meteor-intercom#31]
IntercomClient is undefined.  Is this relating to #25 ?

My intercom Messenger is up and running OK

**Comment by shelkie** [comment]
Have you added your Intercom account info to Meteor.settings?

Meteor.settings.public.intercom
Meteor.settings.public.intercom.id
Meteor.settings.intercom.apikey *(not public)*

**Comment by justan0therdave** [comment]
Yes all settings present and correct - my Messenger is working correctly.

**Comment by dburles** [comment]
It seems IntercomClient wasn't exporting the instance correctly, this is now fixed with 1.4.0 though with a slightly different usage:

```js
const client = IntercomClient();

// Create an event for a user
client.events.create({
  event_name: 'Test event',
  created_at: Math.floor(new Date().getTime() / 1000),
  user_id: '<Insert meteor user id>'
}, function (d) {
  console.log(d);
});
```
